integration/integrate-with-image-vulnerability-scanners.adoc

```diff
@@ -32,22 +32,23 @@ This enhanced support gives you greater flexibility and choice in managing your
 
 You can set up {product-title-short} to obtain image vulnerability data from the following commercial container image vulnerability scanners:
 
-* {product-title-short} Scanner (recommended)
+* {product-title-short} Scanner V4: Beginning with release 4.4, a new scanner is introduced that is built on the {product-title-short} StackRox and Clair scanners. The new scanner, Scanner V4, is the default scanner for language and OS-specific image components. You do not have to create an integration to use this scanner.
```

Comment:
Not sure if I understand this correctly (and if this is ready for review yet), but Scanner V4, as I understand it, *will become* the default scanner in a future release. With 4.4 that cannot be the case, since it is not even installed by default.

Reply:
Oops, this is leftover from when it was going to be the default, so I need to update this. I still have some more things to fix before asking for an official review - ty for catching this one!